extensions/.import-restrictions

```diff
@@ -23,4 +23,6 @@ rules:
   - github.com/gardener/gardener/pkg/logger
   - github.com/gardener/gardener/pkg/resourcemanager/controller/garbagecollector/references
   - github.com/gardener/gardener/pkg/utils
-  - github.com/gardener/gardener/third_party/mock
\ No newline at end of file
+  - github.com/gardener/gardener/third_party/mock
+  # TODO(dimityrmirchev): Remove the below rule when the ShootManagedIssuer feature gate is promoted to GA.
+  - github.com/gardener/gardener/pkg/features
```

Comment:
Why is this needed here? 😕

Reply:
Seems to not be needed after the latest rebase https://github.com/gardener/gardener/pull/9489/commits/e42876ef942588dffab89201ccc06205d605b84b